add deploy-1 docs (#2581)
wso2/docs-is: en/docs/deploy/get-started/customize-the-management-console.md

1.  **UI inherited from WSO2 Carbon platform** 

    The bundle, `org.wso2.carbon.ui_<version-number>.jar` located in `<IS_HOME>/repository/components/ plugins/`, contains the templates and styles (css files) for the Management Console. The `<version-number>` is the version of the Carbon Kernel that WSO2 Identity Server is built on. This 
    bundle is responsible for the overall look and feel of the entire Carbon platform.
    
2.  **Unique UI of each product** 

    Each Carbon product (that is built on Carbon kernel) has another style bundle, 

>     `org.wso2. identity.styles_<version-number>.jar` located in `<IS_HOME>/repository/components/plugins/`. This contains all the styles and images that 

Reviewer: ```suggestion
    `org.wso2.identity.styles_<version-number>.jar` located in `<IS_HOME>/repository/components/plugins/`. This contains all the styles and images that 
```

### Change the product specific styles and images

The styles and images unique to WSO2 Identity Server Management Console are located in the

> `org.wso2.identity. styles_<version_number>.jar` file.

Reviewer: ```suggestion
`org.wso2.identity.styles_<version_number>.jar` file.
```

### Change the product specific styles and images

> The styles and images unique to your product are located in the

Reviewer: ```suggestion
The styles and images unique to WSO2 Identity Server Management Console are located in the
```

> `org.wso2.<product-name>. styles_<version_number>.jar` file.

Reviewer: ```suggestion
`org.wso2.identity.styles_<version-number>.jar` file.
```

Reviewer: saw that throughout this doc the placeholder `<product-name>` has been used. let's change it to identity since we are only focusing on the identity server as the product
